recipe/meta.yaml

```diff
@@ -12,7 +12,7 @@ package:
 
 source:
   url: https://www.sqlite.org/{{ year }}/sqlite-autoconf-{{ version_coded }}.tar.gz
-  sha256: e7974aa1430bad690a5e9f79a6ee5c8492ada8269dc675875ad0fb747d7cada4
+  sha256: 61f2dd93a2e38c33468b7125967c3218bf9f4dd8365def6025e314f905dc942e
   patches:
     - expose_symbols.patch  # [win]
```

Comment:
So the 0001- patch is not used, and a similar non-numbered patch is used?

Reply:
Correct, that's a change coming from conda forge on a previous release.
It's the same patch, however. Before it used to be applied to all architecture and contained #ifdef _WIN32.
Now it is only applied to windows and does not contain #ifdef _WIN32.